smsdk/client.py

```diff
@@ -493,24 +567,7 @@ def get_lines(self, **kwargs):
         )
         return lines(self.session, base_url).get_lines(**kwargs)
 
-    one_day_relative = {
-        "time_type": "relative",
-        "relative_start": 1,
-        "relative_unit": "day",
-        "ctime_tz": "America/Los_Angeles",
-    }
-
-    def get_line_data(
-        self,
-        assets,
-        fields=[],
-        time_selection=one_day_relative,
-        asset_time_offset={},
-        filters=[],
-        limit=400,
-        offset=0,
-        **kwargs,
-    ):
+    def get_line_data(self, *args, **kwargs):
```

Comment:
=None the args

Reply:
This is the standard way of defining positional arguments. If there are no positional arguments passed, it will be come None. No need to set it explicitly.